docs/tutorials/dronecan_px4_flight_controller.rst

```diff
@@ -0,0 +1,449 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _dronecan_fc_tutorial:
+
+****************************************************
+DroneCAN Integration with a PX4 Flight Controller
+****************************************************
+This document provides a general procedure for integrating a Vertiq module with a PX4 flight controller using DroneCAN, also known as UAVCANv0. 
+
+Details on the DroneCAN protocol can be found on the `DroneCAN specification <https://dronecan.github.io/Specification/1._Introduction/>`_. For information on DroneCAN support on 
+Vertiq modules, refer to the :ref:`manual_dronecan` section of the Feature Reference Manual.
+
+Module Configuration and Enumeration
+=====================================
+The first step in the process of setting up a Vertiq module is configuring it through the IQ Control Center. The sections below describe these parameters and give suggestions 
+on how to set them up.
+
+Bitrate
+********
+Currently on Vertiq modules, the bitrate for DroneCAN defaults to 500000 bit/s. It is possible to change this bitrate using the *DroneCAN Bitrate* parameter in 
+IQ Control Center as shown below, or through a :ref:`DroneCAN configuration parameter <dronecan_bitrate_parameter>`. Whatever the bitrate on the module is set 
+to must match the bitrate of the flight controller.
+
+.. figure:: ../_static/tutorial_images/dronecan_px4_tutorial/control_center_bitrate_parameter.png
+    :align: center
+    :width: 60%
+    :alt: DroneCAN Bitrate
+
+    DroneCAN Bitrate Parameter in IQ Control Center
+
+For the purposes of this example, the bitrate is assumed to be 500000 bit/s.
+
+Mode, Direction, and Limits
+****************************
+Descriptions of some important parameters for the module’s direction and mode and how to change them through the IQ Control Center can be found in the :ref:`Motor Configuration section 
+of the tutorial for flight controller integration using hobby protocols <hobby_fc_tutorial_motor_configuration>`. That tutorial is focused on using hobby protocols, but the details on configuring 
+the module are still accurate for DroneCAN.
+
+Node ID
+********
+The next important setting is the node ID. This is the ID the motor will use to identify itself on the DroneCAN bus. **The node ID must be unique for each module, and none of the 
+module’s should use the same node ID as the flight controller.** Typically PX4 flight controllers will use node ID 1 by default, so it is best to avoid that ID, and ID 0 is reserved. You should 

```

Comment:
before you said that Node ID 0 was reserved for the flight controller. Is that not true?

Reply:
Where did I say that? I can't find that on this page. ID 0 is not reserved for the flight controller, its reserved for dynamic node allocation stuff

Reply:
"ID 0 is typically reserved for the flight controller." in the DroneCAN docs: https://iqmotion.readthedocs.io/en/transfer-existing-dronecan-docs/manual/manual_dronecan.html